### PR TITLE
Folder dumb main on mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
This is a first pass at the main folders page in mobile. Likely we'll have to make this a ListView but for now this works

### CODE REVIEW

File: shared/folders/row.native.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+const Avatars = ({styles, users, isPublic}) => {
+  const contents = users.length === 1 || users.length === 2
+      ? <Avatar size={32} username={users[users.length - 1].username} />
+      : <Icon type={styles.groupIcon} />
+
+  if (isPublic) {
+    return <Box style={styles.avatarContainer}>{contents}</Box>
+  }
+
+  // $FlowIssue doesn't like images
```
Comment (chrisnojima) [inline shared/folders/row.native.js]: flow tries to resolve this like a js file so it gets confused, dunno if ther's some larger workaround for this

**Comment by chrisnojima** [comment]
background image working now

**Comment by chromakode** [comment]
If the background image tiling persists to be a problem, it might be convenient to write a webpack loader that automates it for us.

File: shared/folders/list.native.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      </Box>}
+      {showIgnored && ignored.map((i, idx) => (
+        <Row
+          key={rowKey(i.users)}
+          {...i}
+          users={i.users}
+          isPublic={isPublic}
+          ignored
+          isFirst={!idx} />
+        ))}
```
Comment (chromakode) [inline shared/folders/list.native.js]: Extra indent here?

File: shared/folders/list.native.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    return (
+      <Box style={stylesContainer}>
+        {this.props.tlfs && this.props.tlfs.map((t, idx) => (
+          <Row
+            key={rowKey(t.users)}
+            {...t}
+            isPublic={this.props.isPublic}
+            ignored={false}
+            isFirst={!idx} />
+          ))}
```
Comment (chromakode) [inline shared/folders/list.native.js]: and here?

File: shared/local-debug.native.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ if (__DEV__ && true) {
   config.printOutstandingRPCs = true
   config.reactPerf = false
   config.overrideLoggedInTab = Tabs.moreTab
-  config.dumbFilter = ''
+  config.dumbFilter = 'tlf'
```
Comment (chromakode) [inline shared/local-debug.native.js]: Was this intended to make it into the pull req?

**Comment by chromakode** [comment]
:+1:

File: shared/folders/list.native.js (hunk 2 above)
Comment (chrisnojima) [inline shared/folders/list.native.js]: does your eslint complain?

**Comment by MarcoPolo** [comment]
:shipit:

**Pull request merged**